client/my-sites/site-settings/date-time-format/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import DateTimeFormatOptions from './date-time-format-options';
+import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import { getSelectedSite } from 'state/ui/selectors';
+
+export const DateTimeFormat = ( { translate, site } ) => {
+	const goBack = () => {
+		page( '/settings/general/' + site.slug );
+	};
```

Comment:
ping @aduth and @ockham for internal use of `page()` for routing - nothing significant, just noting another use of it

for new code, should we consider creating a new navigation Redux action whose middleware simply calls `page()` so we can start to phase out the imperative navigation?